### DISCUSSION TITLE
Crashlytics didn't work

### DISCUSSION THREAD

**Discussion by OSMAnand** [invertase/react-native-firebase#5634] [Q&A]
Hi, 
I am using :
-React native version : 0.59
-react-native-firebase": "^5.5.0",
I followed [official document](https://v5.rnfirebase.io/docs/v5.x.x/crashlytics/android) to implement crashlytics.
but somehow its not logging crash on firebase console.
I did check my device log and found its saying : **Crashlytics build id is missing** 

<img width="1066" alt="4siUE" src="https://user-images.githubusercontent.com/65129667/130391444-69651c00-3115-499e-8a1e-85229ce6d63e.png">

Here is my **android/app/build.gradle**
<img width="1066" alt="builgradle" src="https://user-images.githubusercontent.com/65129667/130392836-808bdfe5-6045-4c49-b2b3-da29bbc34eba.png">

and here is my **android/build.gradle**
<img width="950" alt="Screenshot 2021-08-23 at 10 31 33 AM" src="https://user-images.githubusercontent.com/65129667/130392953-65970536-507a-4588-b74b-ee597ecff3a9.png">

**Comment by mikehardy** [comment]
>  -React native version : 0.59
> -react-native-firebase": "^5.5.0",

Upgrade to current stable. These versions are not supported at all, in any way, sorry

**Reply by OSMAnand** [reply]
@mikehardy For now I can not upgrade RN version and I implemented firebase analytics with he same "react-native-firebase": "^5.5.0"," and its working fine. I just want to implement crashlytics.

**Reply by mikehardy** [reply]
Sorry, these versions are simply not supported, they are too old. You will be on your own, I will be surprised if you can get it to work - I strongly strongly suggest you carve out the time to cross the barrier from react-native 0.59 to current and react-native-firebase v5 to current. It's the only investment of time that pays dividends / has a future. Any time spent on these versions is wasted.
